### ISSUE TITLE
Right click menu options order is unstable/indeterministic

### ISSUE THREAD

**Issue by huchenlei** [Comfy-Org/ComfyUI_frontend#103]
https://github.com/comfyanonymous/ComfyUI/issues/3883

**Comment by mcmonkey4eva** [comment]
Is this still an issue?

**Comment by huchenlei** [comment]
Yes, it is still an issue. I think the ultimate solution is that we write the context menu building logic here instead of relying on litegraph's logic.

**Comment by Myestery** [comment]
We have fixed this by standardizing the extension's context menu API.
